tests/unit/panther_analysis_tool/test_main.py

```diff
@@ -46,6 +46,34 @@ def test_rules_from_folder(self):
         assert_equal(return_code, 0)
         assert_equal(len(invalid_specs), 0)
 
+    def test_rules_from_current_dir(self):
+        # This is a work around to test running tool against current directory 
```

Comment:
A note about the testing method:
Using `pyfakefs` in conjunction with `importlib` causes some odd behavior that may be related to how things are patched.  I tried to add a test using the pyfakefs in place, but ran into errors.  Tests attempted:

1. Changing current working directing into sub dir within the fakefs and running the test from that context:
`os.chdir(/test/fixtures) -> run test -> os.chdir(original_working_dir)` This results in: “file or directory does not exist” for all the *.py files (seems to read the *.yml files)
2. Adding the files to the fakefs by using add_real_file  into the current working directory. This also results in: “file or directory does not exist” for the *.py files copied into the directory even though testing via `assert_true` (shown below) do not through an exception.  (again, seems to read the *.yml file)
```
self.fs.add_real_file(self.fixture_path+'example_policy.py',target_path="/example_policy.py")
self.fs.add_real_file(self.fixture_path+'example_policy.yml',target_path="/example_policy.yml")
assert_true(os.path.exists('./example_policy.yml'))
assert_true(os.path.exists('./example_policy.py'))
```
In both of these cases:
- os.listdir shows that the *.py files do “exist”
- The difference between reading the yml and the py file is that .yml is read in using open and the .py file is read in using importlib methods; which is why I thought perhaps an issue with pyfakefs patching methods. 

Worked around this by pausing the fakefs context to access the host filesystem, and resuming the fakefs for the remaining tests.